Story/RainMeadow.StoryHooks.cs

```diff
@@ -326,16 +329,28 @@ private void RainWorldGame_GoToDeathScreen(On.RainWorldGame.orig_GoToDeathScreen
         {
             if (OnlineManager.lobby != null && OnlineManager.lobby.gameMode is StoryGameMode)
             {
-                if (!OnlineManager.lobby.isOwner)
+                foreach (var playerAvatar in OnlineManager.lobby.playerAvatars.Values)
                 {
-                    if (!OnlineManager.lobby.owner.OutgoingEvents.Any(e => e is RPCEvent rpc && rpc.IsIdentical(RPCs.MovePlayersToDeathScreen))) {
-                        OnlineManager.lobby.owner.InvokeRPC(RPCs.MovePlayersToDeathScreen);
+                    if (playerAvatar.type == (byte)OnlineEntity.EntityId.IdType.none) continue; // not in game
+                    if (playerAvatar.FindEntity(true) is OnlinePhysicalObject opo && opo.apo is AbstractCreature ac)
+                    {
+                        if (ac.state.alive)
+                            RainMeadow.Debug($"still alive: {playerAvatar}");
                     }
                 }
-                else
+
+                if (OnlineManager.lobby.isOwner)
```

Comment:
Wouldn't be bad to check for outgoing rpc, you can prob ignore this comment though because I added the check to prove that this wasn't causing the starving issue

Reply:
oh i changed it such that clients can't invoke `MovePlayersToDeathScreen` on the host, only the host can trigger it themselves

will likely have to be changed anyways for the spectator mode PR

Reply:
Right, it should be impossible for the host to call moveplayerstodeathscreen multiple times now. So you can safely ignore the check